wurst/_handles/Destructable.wurst

```diff
@@ -69,6 +69,78 @@ public function destructable.setOccluderHeight(real height)
 public function destructable.getName() returns string
 	return GetDestructableName(this)
 
+// Extensions
+/** used to open a destructable gate*/
+public function destructable.gateOpen()
```

Comment:
I'm not sure this belongs here. Is it just a wrapper for `kill`?

It may be better to wrap `destructable` in a tuple type for gates `tuple gate(destructable killable)`